.changeset/big-beans-appear.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react-brand': minor
+---
+
+Decreased the default heading size in `River` component. Use `size="4"` and value to restore the previous text size, if needed.
```

Comment:
This might be a typo

```suggestion
Decreased the default heading size in `River` component. Use `size="4"` to restore the previous text size, if needed.
```

Reply:
Thanks Josh, i left the word "prop" _and value_ out 🤦. But I like your version better so will go with that 👍